src/OGL-UK-3.0.xml

```diff
@@ -0,0 +1,65 @@
+                <item>'Information Provider' means the person or organisation providing the Information under this licence.</item>
+                <item>'Licensor' means any Information Provider which has the authority to offer Information under the terms of this licence or the Keeper of Public Records, who has the authority to offer Information subject to Crown copyright and Crown database rights and Information subject to copyright and database right that has been assigned to or acquired by the Crown, under the terms of this licence.</item>
+                <item>'Use' means doing any act which is restricted by copyright or database right, whether in the original medium or in any other medium, and includes without limitation distributing, copying, adapting, modifying as may be technically necessary to use it in a different mode or format.</item>
+                <item>'You', 'you' and 'your' means the natural or legal person, or body of persons corporate or incorporate, acquiring rights in the Information (whether the Information is obtained directly from the Licensor or otherwise) under this licence.</item>
+            </list>
+            <p>About the Open Government Licence</p>
+            <p>The National Archives has developed this licence as a tool to enable Information Providers in the public sector to license the use and re-use of their Information under a common open licence. The National Archives invites public sector bodies owning their own copyright and database rights to permit the use of their Information under this licence.</p>
+            <p>The Keeper of the Public Records has authority to license Information subject to copyright and database right owned by the Crown. The extent of the offer to license this Information under the terms of this licence is set out in the UK Government Licensing Framework.</p>
+            <p>This is version 3.0 of the Open Government Licence. The National Archives may, from time to time, issue new versions of the Open Government Licence. If you are already using Information under a previous version of the Open Government Licence, the terms of that licence will continue to apply.</p>
+            <p>These terms are compatible with the Creative Commons Attribution License 4.0 and the Open Data Commons Attribution License, both of which license copyright and database rights. This means that when the Information is adapted and licensed under either of those licences, you automatically satisfy the conditions of the OGL when you comply with the other licence. The OGLv3.0 is Open Definition compliant.</p>
+            <p>Further context, best practice and guidance can be found in the UK Government Licensing Framework section on The National Archives website.</p>
+        </text>
+    </license>
+</SPDXLicenseCollection>
```

Comment:
Looks like it's complaining that there's no newline at the end of the file. I don't know whether that matters here...?